helper/helper.py

```diff
@@ -3,13 +3,13 @@
 from sqlalchemy import MetaData, create_engine
 from sqlalchemy.orm import sessionmaker
 
-DB_USER = 'root'
+
 DB_PASSWORD = 'root' # noqa
-DB_URL = 'localhost:5432/perfectfit'
 
 
-def get_db_session():
-    db_loc = 'postgresql://' + DB_USER + ":" + DB_PASSWORD + "@" + DB_URL
+def get_db_session(db_host = 'localhost', db_user = 'root'):
```

Comment:
Given what you wrote in the README, should we be using a different default for `db_host` here?

Reply:
@raar1 I am not sure how many people work on Linux/Mac/Windows. The host.docker.internal works only on Mac and Windows, not on Linux. Bouke and I are in Windows, Cunliang on Mac, but the rest on Linux I think.